src/misc/like.ts

```diff
@@ -0,0 +1,243 @@
+import { DatabaseEntry } from '../common/datas/database';
+import { dataType, dataHolder } from '../common/datas/holder';
+import { dataView } from '../common/datas/view';
+import * as yup from 'yup';
+import { Bukkit, OfflinePlayer } from 'org.bukkit';
+import { UUID } from 'java.util';
+import { isAdminAccount } from '../common/helpers/player';
+import { Player } from 'org.bukkit.entity';
+import { CommandSender } from 'org.bukkit.command';
+
+const playerLikesDataType = dataType('playerLikesData', {
```

Comment:
```suggestion
const LikedPlayers = dataType('likedPlayers', {
```
Use a capital letter to distinguish data types instead of postfixing 'type'. Also, mind the naming: figuring out the difference between e.g. PlayerLikesData and PlayerLikes without reading code is no fun.